distributed/utils.py

```diff
@@ -1464,7 +1464,7 @@ def _repr_html_(self):
         return "\n".join(summaries)
 
 
-def cli_keywords(d: dict, cls=None):
+def cli_keywords(d: dict, cls=None, cmd=None):
```

Comment:
I'm not sure I agree with the changes here.  This function used to do a very simple thing, now it does that thing and a more complex thing that I think was tacked on just for this one use case.

Can you say a bit more about what you're trying to accomplish, and then maybe we can find another approach.

Reply:
Oh, I think that you're trying to change the worker_module string, like `distributed.cli.dask_worker` to an actual module?

I recommend that we make a separate convenience function for this other than `cli_keywords`, and then maybe we can use it in the `ssh2.py::Worker` class instead?  Something like that might be a cleaner separation.

Also, is it easy to add a test for this functionality?  That might make it easier for me to see what you're trying to do here.

Reply:
I'm not sure I understand your comments. But a short explanation is that `dask_cuda_worker` is a specialization of `Worker`, it takes other parameters that `Worker` doesn't know about (such as `--cpu-affinity`), but only the `dask-cuda-worker` executable knows of. I'm not sure I can find a self-contained example that doesn't depend on `dask-cuda` at the moment, but I'll look for one.

Reply:
Ok, I extended the existing tests. To clarify `nprocs` is an option from the `cli/dask_worker.py`, so `SSHCluster` would have failed to start with that option before. Also added a test to ensure unimplemented options raise an exception.